docs/GettingStarted.md

```diff
@@ -3,190 +3,425 @@
+| MRI   | https://www.ruby-lang.org/ | 1.9.1   | Experimental |
+|       |                            | 1.9.3   | Full         |
+|       |                            | 2.0     | Full         |
+|       |                            | 2.1     | Full         |
+|       |                            | 2.2     | Full         |
+|       |                            | 2.3     | Full         |
+|       |                            | 2.4     | Full         |
+| JRuby | http://jruby.org/          | 9.1.5   | Experimental |
+
+*Full* support indicates all tracer features are available.
+
+*Experimental* indicates most features should be available, but unverified.
+
+**Supported web servers**:
+
+| Type      | Documentation                     | Version      | Support type |
+| --------- | --------------------------------- | ------------ | ------------ |
+| Puma      | http://puma.io/                   | 2.16+ / 3.6+ | Full         |
+| Unicorn   | https://bogomips.org/unicorn/     | 4.8+ / 5.1+  | Full         |
+| Passenger | https://www.phusionpassenger.com/ | 5.0+         | Full         |
+
+## Installation
+
+The following steps will help you quickly start tracing your Ruby application.
+
+### Setup the Datadog Agent
+
+The Ruby APM tracer sends trace data through the Datadog Agent.
+
+[Install and configure the Datadog Agent](https://docs.datadoghq.com/tracing/setup), see additional documentation for [tracing Docker applications](https://docs.datadoghq.com/tracing/setup/docker/).
+
+### Quickstart for Rails applications
+
+1. Add the `ddtrace` gem to your Gemfile:
+    
+    ```ruby
     source 'https://rubygems.org'
-
-    # tracing gem
     gem 'ddtrace'
+    ```
 
-If you're not using ``Bundler`` to manage your dependencies, you can install ``ddtrace`` with:
-
-    gem install ddtrace
+2. Install the gem with `bundle install`
+3. Create a `config/initializers/datadog.rb` file containing:
 
-We strongly suggest pinning the version of the library you deploy.
-
-## Quickstart
+    ```ruby
+    Datadog.configure do |c|
+      # This will activate auto-instrumentation for Rails
+      c.use :rails
+    end
+    ```
 
-The easiest way to get started with the tracing client is to instrument your web application.
-All configuration is done through ``Datadog.configure`` method. As an
-example, below is a setup that enables auto instrumentation for Rails, Redis and
-Grape, and sets a custom endpoint for the trace agent:
+### Quickstart for Ruby applications
 
-    # config/initializers/datadog-tracer.rb
+1. Install the gem with `gem install ddtrace`
+2. Include the Ruby application with:
 
+    ```ruby
+    require 'ddtrace'
     Datadog.configure do |c|
-      c.tracer hostname: 'trace-agent.local'
-      c.use :rails
-      c.use :grape
-      c.use :redis, service_name: 'cache'
+      # Configure the tracer here.
+      # Activate integrations, change tracer settings, etc...
+      # By default without additional configuration, nothing will be traced.
```

Comment:
If I'm reading the doc, I would like to add additional instrumentation. Unfortunately here I only know about `:rails` and I need to find other symbols (i.e. grape). Would be better to add a link here so that people can reach the integrations list.

Reply:
https://github.com/DataDog/dd-trace-rb/blob/f20db4c45614fab65f9f4d9cdb1932d0a062dcbe/docs/GettingStarted.md#integration-instrumentation

Reply:
This should be addressed.